CONFIG.YAML

```diff
@@ -15,3 +18,30 @@ HMC:
        - "lshwres -r virtualio --rsubtype eth -m SYS --level lpar" #Virtual Ethernet Adapter (VEA) Info
        - "lshwres -r virtualio --rsubtype scsi -m SYS --level lpar" #scsi Adapter Info
        - "lshwres -r virtualio --rsubtype fc -m SYS --level lpar"  #fc Adapter Info
+       - "lssyscfg -r lpar -m SYS -F name,kvm_capable" #To check KVM capable lpars
+       - "lssysconn -r all" # This command lists system connections and includes the FSP/BMC IP address.
+
+IO:
+  COMMANDS:
+    - "lshwres -r io -m SYS --rsubtype slot --filter 'lpar_names=LPAR_NAME'"  # List physical IO
+    - "lshwres -r virtualio --rsubtype fc --level lpar -m SYS -F adapter_type,lpar_name,slot_num,remote_lpar_name,remote_slot_num,wwpns"  # List virtual IOs
+
```

Comment:
ideally it should be part of HMC

Reply:
for USER i feel it's just a command section .. Component mapping can be done based on individual